### DISCUSSION TITLE
Remove topics from messages and replace with Joomla tags.

### DISCUSSION THREAD

**Discussion by genu** [Joomla-Bible-Study/Proclaim#845] [Ideas]
Related to #118 

Absorb the JBS topic system into the new Joomla **Tags** mechanism.

**Comment by bcordis** [comment]
This may be an option to convert them into tags without a problem. Thin linking the tags to the old messages. This must be done as we will soon be moving away from the topics system.

**Comment by tomfuller2** [comment]
Should we tackle this now? How hard/easy would it be? Perhaps this is for after 10.x? Or if not hard, it would simplify our code and we could get rid of the current topics admin and migrate Proclaim topics to Joomla tags.

**Comment by bcordis** [comment]
Ok, looking into this may get into 10.1.0, but may need to hold off till 10.2.0

**Comment by bcordis** [comment]
Moved to 11.x as the release phase due to the massive change in code.
